src/langchain_google_alloydb_pg/alloydb_vectorstore.py

```diff
@@ -731,9 +731,13 @@ def max_marginal_relevance_search_with_score_by_vector(
         return self.engine._run_as_sync(coro)
 
     async def set_maintenance_work_mem(self, num_leaves: int, vector_size: int) -> None:
-        index_memory_required = 50 * num_leaves * vector_size * 4
+        # Required index memory in MB
+        buffer = 1
```

Comment:
Is there a reason for the buffer?

Reply:
The memory required for scann index creation is rounded to the nearest MB. I set this buffer so that:
1. When the memory calculated is less than 1MB, it wouldn't output 0.
2. In case there is any other maintenance memory usage during index creation, add 1 MB to be safe.